analysers/analyser_merge_parking_FR_BNLS

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2014                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item = 8130, id = 1, level = 3, tags = ['merge', 'parking'],
+            title = T_f('{0} parking not integrated', 'BNLS'))
+
```

Comment:
i suppose here i must add item 8131 and 8132 for integration suggestion and update ?

Reply:
No, since you have no ref.

Reply:
ok, so i must add ref now (id is the unique identifier of the car park, issued by the Point d'accès national)

Reply:
euh ok for update but for integration suggestion i need ref ? because i think there's a lot of parking already existing in osm (http://umap.openstreetmap.fr/fr/map/carte-sans-nom_452448#7/46.740/5.076)

Reply:
There is public, stable ref in the source file? So you can map it to a tag, and enable the other part of the merge.

Reply:
yes it seems to be aims "public, stable ref".
i resolve this conversation